docs/features/source-map-support.md

```diff
@@ -37,16 +37,16 @@ module.exports = {
 
 Exceptions are logged into your application error log file.
 
-To check your logs to detect exceptions, you simply have to type:
+To check your logs to detect exceptions, you can type:
 
 ```bash
 pm2 logs main
 ```
 
-Else use [keymetrics.io](https://keymetrics.io/) to have a clean listing and notifications of [new alerts happening](http://docs.keymetrics.io/docs/pages/issues/).
+Otherwise, use [keymetrics.io](https://keymetrics.io/) to have a clean listing and notifications of [new alerts](http://docs.keymetrics.io/docs/pages/issues/).
 
 ### Disable source map support
 
-If you do not want PM2 to automatically support javascript source map you can use the option `--disable-source-map`.
+If you do not want PM2 to automatically support javascript source maps you can use the option `--disable-source-map`.
```

Comment:
> ...use the option `--disable-source-map`.

Not something you've changed, but this option is actually "--disable-source-map-support".

I've [made a pull request to change this](https://github.com/pm2-hive/pm2-hive.github.io/pull/203), but your changes are much more extensive and could replace mine.